### DISCUSSION TITLE
要想使用动态主题色切换功能是不是就必须先将全部组件的css引入，不能按需加载样式了？

### DISCUSSION THREAD

**Discussion by weeksun23** [vueComponent/ant-design-vue#5347] [Q&A]
要想使用动态主题色切换功能是不是就必须先将全部组件的css引入，不能按需加载样式了？

**Comment by geeklibin** [comment]
引入css肯定不好实现换肤。最好引入less：
核心思路是用less js动态改变皮肤变量
请参考：[https://blog.csdn.net/qq873113580/article/details/123008042](https://blog.csdn.net/qq873113580/article/details/123008042)
